test/factories/personal_access_tokens.rb

```diff
@@ -1,8 +1,8 @@
 FactoryBot.define do
   factory :personal_access_token do
     sequence(:name) {|n| "Personal Access Token #{n}" }
-    token { Digest::SHA1.hexdigest(SecureRandom.urlsafe_base64(nil, false)) }
     revoked { false }
+    token { BCrypt::Engine.hash_secret('OSKoHLYqOj0AQd5dnyb8sw', PersonalAccessToken.token_salt_bcrypt(user.id)) }
```

Comment:
Where does `user` come from?

Reply:
Whoops that was the last rebase, let me take a look. But please keep going!

Reply:
Fixed, let's use a constant token for mock.